### DISCUSSION TITLE
Feature Request - Custom Species

### DISCUSSION THREAD

**Discussion by ninjew42** [bcholmes/StarTrek2d20#117] [Ideas]
When making Starships there's a custom spaceframe option with rules available thanks to several sources. When making a supporting or main character currently we would need to choose Human, Hologram, or a similar species that allows choice of Attributes, override the talents, etc and then modify the talents after if there's a custom talent for the species or if it's a villain who like Khan have the ability to be threatening and gains Augment bonuses.
It would be awesome if we could have a Custom Species choice that opened the menu into choosing a name for the species, attributes/bonuses could be selected and Talents could be either selected or typed in along with any natural attacks (such as claws) or natural resistances (like having +3 resistance as a Borg)
It would also be putting the alien creation rules in the Sciences Division and Core book to use.

**Comment by bcholmes** [comment]
Yeah, definitely a needed function.

**Comment by Felderburg** [comment]
The custom species would also need a field for entering its trait.

**Reply by bcholmes** [reply]
Say more on that. Usually the species trait is the same name as the species. Are you thinking of something more?

**Reply by Felderburg** [reply]
I meant the paragraph that describe the trait/species in detail. E.g. Andorians have "TRAIT: Andorian. This trait may reduce the Difficulty of Tasks to resist extreme cold, or Tasks impacted by extremely low temperatures. Their antennae..."

**Reply by 1001100x02** [reply]
Also, when exporting the custom species to PDF, it ends up overriding the Species name with your Rank
